substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/util/Timer.java

```diff
@@ -78,10 +78,11 @@ private void print(long time) {
         if (prefix != null) {
             // Add the PID to further disambiguate concurrent builds of images with the same name
             String pid = GraalServices.getExecutionID();
-            System.out.format("[%s:%s] %12s: %,10.2f ms\n", prefix, pid, name, time / 1000000d);
-        } else {
-            System.out.format("%12s: %,10.2f ms\n", name, time / 1000000d);
+            System.out.format("[%s:%s] ", prefix, pid);
         }
+        final double heap = Runtime.getRuntime().totalMemory() / 1024.0 / 1024.0 / 1024.0;
+        System.out.format("%12s: %,10.2f ms, %,5.2f GB", name, time / 1000000d, heap);
+        System.out.println();
```

Comment:
The output should be written in a single `System.out.format` call to avoid interleaving of concurrent native image executions. Also, please replace `\n` in the format string with `%n`.

Reply:
Thanks - I've reformatted so it's a single `format` call to make it concurrency-safe.

However the `\n` was in the _existing_ code - not mine. My code used `println` (and now uses `%n`) - I was never using `\n`.

Reply:
Yes, I should have pointed out that my request was partially to fix up existing code.

Reply:
Great it's `%n` now so all done.